packages/cli/src/commands/deploy/index.ts

```diff
@@ -181,6 +181,21 @@ export default async (client: Client) => {
     );
   }
 
+  // build `--prebuilt`
+  if (argv['--prebuilt']) {
+    const prebuiltExists = await fs.pathExists(join(path, '.vercel/output'));
+    if (!prebuiltExists) {
+      error(
+        `The ${param(
+          '--prebuilt'
+        )} option was used, but no prebuilt deploy found in ".vercel/output". Run ${getCommandName(
```

Comment:
"no prebuilt deploy found" should probably be "no prebuilt output found"

Reply:
Agreed! https://github.com/vercel/vercel/pull/7702